src/repositories/request-repository.ts

```diff
@@ -152,7 +157,14 @@ const findRequestsToAnchorForStreams = (
   streamIds: string[],
   now: Date
 ): Promise<Array<Request>> =>
-  findRequestsToAnchor(connection, now).whereIn('streamId', streamIds).orderBy('createdAt', 'asc')
+  findRequestsToAnchor(connection, now)
+    .whereIn('streamId', streamIds)
+    .orderByRaw(
```

Comment:
why do we need this raw thing based on the streamids array?  Could we just orderBy stream_id?

Reply:
ping

Reply:
We do this because the streamIds array is sorted based on the earliest request for each streamId (ascending). Since we want to anchor expiring requests firsts we have to do this weird raw thingy. 

If we were to order by stream_id. We may not pick up expiring requests.

Reply:
can we add a comment explaining that?